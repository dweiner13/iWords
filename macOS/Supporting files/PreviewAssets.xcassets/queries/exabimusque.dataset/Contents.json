{
  "data" : [
    {
      "filename" : "exabimusque.txt",
      "idiom" : "universal"
    }
  ],
  "info" : {
    "author" : "xcode",
    "version" : 1
  }
}
