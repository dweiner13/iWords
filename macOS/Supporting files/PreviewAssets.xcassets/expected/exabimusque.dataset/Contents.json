{
  "data" : [
    {
      "filename" : "exabimusque.json",
      "idiom" : "universal"
    }
  ],
  "info" : {
    "author" : "xcode",
    "version" : 1
  }
}
